bundles/org.openhab.core/src/main/java/org/openhab/core/util/ColorUtil.java

```diff
@@ -205,6 +203,14 @@ public static HSBType rgbToHsb(int[] rgb) throws IllegalArgumentException {
                 convertByteToColorPercent(rgb[2]) });
     }
 
+    /**
+     * Transform <a href="https://en.wikipedia.org/wiki/SRGB">sRGB</a> color format to
+     * <a href="https://en.wikipedia.org/wiki/HSL_and_HSV">HSV</a> based {@link HSBType}.
+     *
+     * @param rgb PercentType[] array of length 3, comprising integer or real values, constrained to 0-100.
```

Comment:
```suggestion
     * @param rgb {@link PercentType} array of length 3
```

`PercentType` is always 0-100, you can't set any other values.

Reply:
Ok

Reply:
But, I don't think there is any reason why the Java doc should not say that.